cli/sawtooth_cli/main.py

```diff
@@ -93,7 +95,11 @@ def create_parser(prog_name):
     add_admin_parser(subparsers, parent_parser)
     add_config_parser(subparsers, parent_parser)
     add_block_parser(subparsers, parent_parser)
+<<<<<<< 8a234e6efe39c7de9acfd8f30978e795f0d7e38c
```

Comment:
I think the intent of the change request is to fix that particular commit, not just the code.